app/i18n/en/conf.php

```diff
@@ -149,6 +149,10 @@
 		'image_url' => 'Image URL',
 		'name' => 'Name',
 		'no_filter' => 'No filter',
+		'no_queries' => array(
+			'_' => 'No user queries are safed yet.',
```

Comment:
```suggestion
			'_' => 'No user queries are saved yet.',
```

I believe you meant *saved* ?